distributed/worker.py

```diff
@@ -3144,18 +3203,11 @@ def update_who_has(
                         )
                         # Do not mutate the input dict. That's rude
                         workers = set(workers) - {self.address}
-                    dep_ts = self.tasks[dep]
-                    if dep_ts.state in FETCH_INTENDED:
```

Comment:
You removed the only use of this constant; I think you should remove the constant declaration too